src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/V2Scripts.scala

```diff
@@ -249,5 +249,6 @@ object V2Scripts {
     OO_gitlab_373_ZahlungsEingang_teilnehmer_nummer.scripts ++
     OO_distribution_date_time_should_not_consider_time.scripts ++
     Operations465_EnlargeKundeBez.scripts ++
-    OO411_adding_person_contact_permission.scripts
+    OO411_adding_person_contact_permission.scripts ++
+    OO86_EnlargeLoginMessage.scripts
```

Comment:
The naming pattern of the scripts seems to have diverged over time. I suggest deciding on one style for the future or even rename `OO86_EnlargeLoginMessage` to `OO86_increase_login_message_size`

Reply:
done